src/components/ImageView/ShowImageModal.vue

```diff
@@ -11,7 +11,7 @@
 		@previous="showPreviousImage"
 		@close="$emit('close')">
 		<div class="modal__content">
-			<img :src="currentImage.source">
+			<img :src="currentImage.fullUrl || currentImage.previewUrl">
```

Comment:
Shall we rather prefer to load the previewUrl if present?

Reply:
Well, given that this is a fullscreen view of the image I thought that `fullUrl` is the better option here, no? Or do you think that we should use `previewUrl` for the fullscreen modal view? In any case the alternative can be removed as both are always set.

Reply:
I think the preview should be big enough and the viewer app also always uses the preview iirc.

Reply:
Alright. Changed now.